build.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/sh
 #
-# SPDX-License-Identifier: AGPL-3.0+
-#
 # Copyright (C) 2018 Ultimaker B.V.
 # Copyright (C) 2018 Olliver Schinagl <oliver@schinagl.nl>
 #
 
-set -eu
+if [ "${MAKEFLAGS}" == "" ]; then
+    echo -e -n "\e[1m"
+    echo "Makeflags not set, hint, to speed up compilation time, increase the number of jobs. For example:"
+    echo "MAKEFLAGS='-j 4' ${0}"
```

Comment:
Why are the number of jobs now locked to 4 instead of number of processors?

Reply:
They are not locked, this is just a printout, but good point. Removed the whole block to simplify, since it is a build of a few seconds.